packages/client/src/generation/utils/buildInjectableEdgeEnv.ts

```diff
@@ -10,22 +7,21 @@ type LoadedEnv = {
 }
 
 /**
- * Builds an inline environment for the data proxy client. This is useful
+ * Builds an injectable environment for the data proxy edge client. It's useful
  * because it is designed to run in browser-like environments where things like
- * `fs`, `process.env`, and .env file loading are not available. The inline env
- * is the default fallback when `tryLoadEnvs` wasn't called by the client.
- * @see {@link declareInlineEnv}
- * @param dataProxy
+ * `fs`, `process.env`, and .env file loading are not available. The injectable
+ * env is the default fallback when `tryLoadEnvs` wasn't called by the client.
+ * @see {@link declareInjectableEdgeEnv}
+ * @param edge
  * @param datasources
- * @param envPaths
  * @returns
  */
-export function buildInlineEnv(dataProxy: boolean | undefined, datasources: InternalDatasource[], envPaths: EnvPaths) {
-  if (dataProxy === true) {
+export function buildInjectableEdgeEnv(edge: boolean | undefined, datasources: InternalDatasource[]) {
+  if (edge === true) {
     const envVarNames = getSelectedEnvVarNames(datasources)
-    const loadedEnv = loadSelectedEnvVars(envPaths, envVarNames)
+    const loadedEnv = getEmptyEnvObjectForVars(envVarNames)
```

Comment:
Did I understand the underlying code (and comments) correctly that this now somethat that makes sure we have fallbacks for all the used env vars - and the fallback is `undefined`? Why can we not just do anything here and still have the same result?

Reply:
Ah this comment here clarified some more: https://github.com/prisma/prisma/pull/13512/files#diff-09784fc0113a735572cff48990efa35434fb6ec6ec8876845ec9cf423642e5fdR240-R242

Reply:
Will add the correct comments in the finalization PR.